azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/configuration/ChatCompositeConfiguration.kt

```diff
@@ -1,49 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.android.communication.ui.chat.configuration
-
-import com.azure.android.communication.ui.chat.ChatCompositeException
-import com.azure.android.communication.ui.chat.models.ChatCompositeLocalizationOptions
-
-internal class ChatCompositeConfiguration {
```

Comment:
Adam, currently, configuration logic is in sync with calling. would it be possible to know the reason to change pattern?